recipes/libftdi/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def imports(self):
+        self.copy("*.so", dst="libftdi")
+        self.copy("*.a", dst="libftdi")
+        self.copy("*.h", dst="libftdi")
+
+    def test(self):
```

Comment:
```suggestion
        cmake.build()

    def test(self):
```

Reply:
#### Fixed

This imports block has been removed.